package.json

```diff
@@ -21,12 +21,12 @@
     "@polymer/iron-icon": "^3.0.0-pre.19",
     "@polymer/iron-iconset-svg": "^3.0.0-pre.19",
     "@polymer/iron-overlay-behavior": "^3.0.0-pre.19",
-    "@polymer/lit-element": "^0.5.1",
+    "@polymer/lit-element": "dev",
     "@polymer/paper-icon-button": "^3.0.0-pre.19",
     "@polymer/paper-spinner": "^3.0.0-pre.19",
     "@polymer/polymer": "3.0.0",
     "@webcomponents/webcomponentsjs": "^2.0.0",
-    "lit-html": "^0.10.0",
+    "lit-html": "dev",
```

Comment:
We should still include it tho since we import the `repeat` directive from lit-html directly (remember that previous user report of that package manager that requires all directly imported dependencies).

Reply:
Oh yeah. That makes sense.